gradlew.bat

```diff
@@ -14,7 +14,7 @@
 @rem limitations under the License.
 @rem
 
-@if "%DEBUG%" == "" @echo off

```

Comment:
changes to this file were made automatically by gradle wrapper when it upgraded itself